php/libraries/ParentIDGenerator.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types=1);
+/**
+ * This file contains a class used to generate ParentIDs for use in LORIS.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   Shen Wang <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntities\Candidate\ParentIDGenerator;
```

Comment:
i guess instead of creating this namespace, this ParentIDGenerator class would `use \LORIS\StudyEntities\Candidate\ParentID`